### ISSUE TITLE
gradlew clean build fails after clone

### ISSUE THREAD

**Issue by adam-wynne** [Azure-Samples/azure-functions-samples-java#35]
All dependencies are installed.  I'm getting the following error:
```shell
./gradlew clean build

> Configure project :
default messager has already been registered

> Task :compileJava FAILED
/home/awynne/dev/repos/github/azure-functions-samples-java/src/main/java/com/functions/CosmosDBTriggerFunction.java:23: error: cannot find symbol
                                               @CosmosDBInput(name = "item", databaseName = "%CosmosDBDatabaseName%", containerName = "ItemsCollectionIn", connection = "AzureWebJobsCosmosDBConnectionString", id = "{docId}") String item,

                               ^
  symbol:   method containerName()
  location: @interface CosmosDBInput
/home/awynne/dev/repos/github/azure-functions-samples-java/src/main/java/com/functions/CosmosDBTriggerFunction.java:23: error: cannot find symbol
                                               @CosmosDBInput(name = "item", databaseName = "%CosmosDBDatabaseName%", containerName = "ItemsCollectionIn", connection = "AzureWebJobsCosmosDBConnectionString", id = "{docId}") String item,

```

**Comment by karianna** [comment]
Works for me using Java 8....

**Comment by karianna** [comment]
Fix in #36
